src/pydantic_pint/quantity.py

```diff
@@ -26,6 +28,7 @@ class PydanticPintQuantity:
             If the field is restricted by dimension, then any unit of that dimension is allowed.
         ureg:
             A custom Pint unit registry.
+            If not specified, the default unit registry `pydantic_pint.registry.DEFAULT_UNIT_REGISTRY` is used.
```

Comment:
DEFAULT_UNIT_REGISTRY does not exist any more

Reply:
Fixed in [9c171d4](https://github.com/tylerh111/pydantic-pint/pull/8/commits/9c171d482566ea346f044cd2b52d16680a2e1bf6)